tests/Formatting/Formatters/FullyQualifiedFacadesTest.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+namespace Tests\Formatting\Formatters;
+
+use PHPUnit\Framework\TestCase;
+use Tighten\TLint\Formatters\FullyQualifiedFacades;
+use Tighten\TLint\TFormat;
+
+class FullyQualifiedFacadesTest extends TestCase
+{
+    /** @test */
+    public function fixes_facade_aliases()
+    {
+        $file = <<<'file'
+<?php
+
+namespace Test;
+
+use DB;
+use Storage;
+
+return true;
+file;
+
+        $formatted = (new TFormat)->format(new FullyQualifiedFacades($file));
+
+        $expected = <<<'file'
+<?php
+
+namespace Test;
+
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Storage;
+
+return true;
+file;
+
+        $this->assertSame($expected, $formatted);
+    }
+
+    /** @test */
+    public function fixes_facade_aliases_when_file_not_namespaced()
+    {
+        $file = <<<'file'
+<?php
+
+use DB;
+use Storage;
+
+return true;
+file;
+
+        $formatted = (new TFormat)->format(new FullyQualifiedFacades($file));
+
+        $expected = <<<'file'
+<?php
+
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Storage;
+
+return true;
+file;
+
+        $this->assertSame($expected, $formatted);
+    }
+
+    /** @test */
+    public function fixes_facade_aliases_mixed_with_valid_use_statements()
+    {
+        $file = <<<'file'
+<?php
+
+namespace Test;
+
+use DB;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Schema;
+use Storage;
+use Tests\TestCase;
+
+return true;
+file;
+
+        $formatted = (new TFormat)->format(new FullyQualifiedFacades($file));
+
+        $expected = <<<'file'
+<?php
+
+namespace Test;
+
+use Illuminate\Support\Facades\DB;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\Storage;
+use Tests\TestCase;
+
+return true;
+file;
+
+        $this->assertSame($expected, $formatted);
+    }
+
+    /** @test */
+    public function fixes_str_and_arr_helpers()
+    {
+        $file = <<<'file'
+<?php
+
+namespace Test;
+
+use Arr;
+use Str;
+
+return true;
+file;
+
+        $formatted = (new TFormat)->format(new FullyQualifiedFacades($file));
+
+        $expected = <<<'file'
+<?php
+
+namespace Test;
+
+use Illuminate\Support\Arr;
+use Illuminate\Support\Str;
+
+return true;
+file;
+
+        $this->assertSame($expected, $formatted);
+    }
+
+    /** @test */
+    public function ignores_grouped_imports()
+    {
+        $file = <<<'file'
+<?php
+
+namespace Test;
+
+use App\{ User, Post };
+
+return true;
+file;
+
+        $formatted = (new TFormat)->format(new FullyQualifiedFacades($file));
+
+        $this->assertSame($file, $formatted);
+    }
+
+    /** @test */
+    public function ignores_unknown_aliases()
+    {
+        $file = <<<'file'
+<?php
+
+namespace Test;
+
+use Shortcut;
+
+return true;
+file;
+
+        $formatted = (new TFormat)->format(new FullyQualifiedFacades($file));
+
+        $this->assertSame($file, $formatted);
+    }
```

Comment:
Can you add another test with the problem we were testing?

```php
<?php

namespace App\Utilities;

class Stuff
{
    public function doStuff()
    {
        return File::otherStuff();
    }
}
```

But `File` here is referencing `App\Utilities\File`, not `Illuminate\Support\Facades\File`.

I don't even know how we'll address this, TBH... hopefully it's possible?

Reply:
Added. Right now we are only looking at and fixing import statements.

This is fine:
```php
<?php

namespace App\Utilities;

class Stuff
{
    public function doStuff()
    {
        return File::otherStuff();
    }
}
``` 

This will trigger an error with the linter and a fix with formatter
```php
<?php

namespace App\Utilities;

use File;

class Stuff
{
    public function doStuff()
    {
        return File::otherStuff();
    }
}
```